pkg/util/pki/csr.go

```diff
@@ -80,6 +92,17 @@ func IPAddressesToString(ipAddresses []net.IP) []string {
 	return ipNames
 }
 
+func URLsToString(uris []*url.URL) []string {
+	var uriStrs []string
+	for _, uri := range uris {
+		if uri != nil {
```

Comment:
Is this ever nil? Is that acceptable input? Might be better to panic if that's the case.. it indicates an issue further up IMO

Reply:
:+1: